server/vulnerabilities/nvd/cpe.go

```diff
@@ -147,6 +150,17 @@ func cpeGeneralSearchQuery(software *fleet.Software) (string, []interface{}, err
 
 	datasets := []*goqu.SelectDataset{search1, search2, search3}
 
+	// 4 - Try vendor/product from bundle identifier, like tld.vendor.product
+	bundleParts := strings.Split(software.BundleIdentifier, ".")
+	if len(bundleParts) >= 3 {
+		search4 := dialect.From(goqu.I("cpe_2").As("c")).
+			Select("c.rowid", "c.product", "c.vendor", "c.deprecated", goqu.L("4 as weight")).
+			Where(
+				goqu.Or(goqu.L("c.vendor = ?", strings.ToLower(bundleParts[1]))), goqu.L("c.product = ?", strings.ToLower(bundleParts[2])),
+			)
+		datasets = append(datasets, search4)
+	}
+
```

Comment:
I added this to handle `com.jetbrains.pycharm`. It should help us match other software as well.

Reply:
This looks great! Thanks for the improvement!